packages/yoastseo/spec/scoring/assessments/inclusiveLanguage/exportInclusiveLanguageConfigSpec.js

```diff
@@ -89,27 +79,12 @@ describe( "Export of the inclusive language configuration", () => {
 		resultLines.unshift( header.join( ";" ) );
 
 		// Set doExport to true to write the results to a temporary file.
-		const doExport = false;
+		const doExport = true;
```

Comment:
Shouldn't this value be set back to false before merge?

Reply:
Yes, thanks!